packages/template-retail-react-app/app/utils/locale.js

```diff
@@ -17,13 +17,13 @@ export const fetchTranslations = async (locale) => {
     const targetLocale =
         typeof window === 'undefined'
             ? process.env.USE_PSEUDOLOCALE === 'true'
-                ? 'en-XB'
+                ? 'en-XA'
```

Comment:
The newest version we are using with formatjs 6.1.1 (from lock file generation) no longer offers en-XB pseudo locale. en-XA is the closest we can use. https://formatjs.io/docs/tooling/cli/#--pseudo-locale-pseudolocale